source/lib/JMSLab/tests/test_build_lyx.py

```diff
@@ -85,11 +88,11 @@ def test_nonexistent_source(self, mock_system):
         # i) Directory doesn't exist
         with self.assertRaises(ExecCallError):
             build_lyx('build/lyx.pdf',
-                      ['bad_dir/lyx_test_file.lyx'], env = {})
+                      ['bad_dir/lyx_test_file.lyx'], env = {'HANDOUT_SFIX': ''})
```

Comment:
All the tests look to involve cases where the user declares a suffix. Should we add some testing for the cases where the user doesn't declare a suffix, to make sure behavior is as intended in that case?

Reply:
I'm on it!

Reply:
@jmshapir Added a test in [e0e1b56](https://github.com/JMSLab/Template/pull/62/commits/e0e1b569e1de020a2b9c4e17073f4c7b8e5f343e) !